app/app.go

```diff
@@ -477,7 +476,11 @@ func New(
 		app.LeverageKeeper,
 	)
 	app.LeverageKeeper.SetTokenHooks(app.OracleKeeper.Hooks())
-	app.LeverageKeeper.SetBondHooks(app.IncentiveKeeper.BondHooks())
+	if Experimental {
```

Comment:
Can move `app.IncentiveKeeper` inside experimental

Reply:
we can initialize the keeper. It doesn't hurt. There is no code running when you initialize the keeper.
It's important to not initialize the module, and not register the hooks nor the begin/end/init blockers

Reply:
Okay